### PR TITLE
Add libs and headers in subpackage in windows

### LINKED ISSUE #36: Windows builds don't include a bunch of extra libraries
Mostly related to #15 , the windows builds don't include libraries/headers to link to.  I'm not sure if there's a way to add to this recipe a labelled build which is essentially `rdkit-dev`

### PR DESCRIPTION
**Pull request by n-elie**
Checklist
* [x] Used a [fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

This implement a subpackage with header and libs for Windows build. Should fix #36.

### CODE REVIEW

**Comment by mcs07** [comment]
This seems like a pretty good solution to me.

@greglandrum - does this look alright to you? The additional `rdkit-dev` package only exists on Windows, and only contains the `lib/*.lib` and `include/*.h` files that were removed in #15 & #16. It depends on the `rdkit` package (which is unchanged), so `conda install rdkit-dev` will install both.

File: recipe/install_rdkit_dev.bat [hunk 1]
```diff
@@ -0,0 +1,11 @@
+echo "Copying libs and headers"
+
+if not exist "%LIBRARY_LIB%" mkdir %LIBRARY_LIB%
+if not exist "%LIBRARY_INC%" mkdir %LIBRARY_INC%
+
+REM copy .lib and .pdb files to LIBRARY_LIB
```
Comment (greglandrum) [inline recipe/install_rdkit_dev.bat:6]: comment says `.pdb` files, but no `.pdb` files are copied

**Comment by greglandrum** [comment]
makes sense to me

File: recipe/install_rdkit_dev.bat (hunk 1 above)
Comment (n-elie) [reply recipe/install_rdkit_dev.bat:6]: My mistake, the install script comes from [another package](https://github.com/nsimakov/wxpython-feedstock/blob/copying_headers_and_libs/recipe/install_wxwidgets.bat) and I forgot to update the comment

**Comment by n-elie** [comment]
@conda-forge-admin, please rerender

**Comment by n-elie** [comment]
Is there anything I can do to improve this?

**Comment by n-elie** [comment]
Thanks for merging

**Pull request merged**